ariadne/utils.py

```diff
@@ -0,0 +1,222 @@
+import torch
+import numpy as np
+import itertools
+import pandas as pd
+from copy import deepcopy
+from ariadne.tracknet_v2.metrics import point_in_ellipse
+import faiss
+import matplotlib.pyplot as plt
+from scipy.interpolate import make_interp_spline, BSpline
+import os
+
+def fix_random_seed(seed):
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+    if torch.cuda.is_available():
+        torch.deterministic = True #torch.backends.cudnn.determenistic
+        torch.benchmark = False #torch.backends.cudnn.benchmark
+
+def cartesian(df1, df2):
```

Comment:
this name is confusing and I saw this function in two or three classes. Please, remove it from them

Reply:
removed